doc/source/whatsnew/v0.22.0.txt

```diff
@@ -83,14 +83,20 @@ Other Enhancements
 Backwards incompatible API changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- :func:`Series.fillna` now raises a ``TypeError`` instead of a ``ValueError`` when passed a list, tuple or DataFrame as a ``value`` (:issue:`18293`)
-- :func:`pandas.DataFrame.merge` no longer casts a ``float`` column to ``object`` when merging on ``int`` and ``float`` columns (:issue:`16572`)
-- The default NA value for :class:`UInt64Index` has changed from 0 to ``NaN``, which impacts methods that mask with NA, such as ``UInt64Index.where()`` (:issue:`18398`)
--
+.. _whatsnew_0220.api_breaking.deps:
 
+Dependencies have increased minimum versions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+We have updated our minimum supported versions of dependencies (:issue:`18613`).
 
+If installed, we now require:
 
+   +--------------+-----------------+----------+
+   | Package      | Minimum Version | Required |
+   +==============+=================+==========+
+   | Cython       | 0.24            |          |
```

Comment:
I don't think we should necessarily be mentioned that prominent (in the table), as it is only a development dependency, not a user dependency

Reply:
agree, this just needs a mention in api breaking, not a sub-section

Reply:
done